ca_tools/scripts/path_printer.py

```diff
@@ -0,0 +1,41 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+import math
+
+import rospy
+from GTSys import GTSys
+from geometry_msgs.msg import Pose, PoseStamped
+from nav_msgs.msg import Path
+
+class pathPrinter():
+
```

Comment:
Add a docstring indicating the purpose of this class.

Reply:
Also, follow the PEP8 naming convetion

Reply:
Addressed

Reply:
Use `CapWords` not `capWords`:
https://www.python.org/dev/peps/pep-0008/#class-names